presto-product-tests/conf/presto/etc/environment-specific-catalogs/singlenode-kerberos-hdfs-no-impersonation-cross-realm/hive.properties

```diff
@@ -0,0 +1,29 @@
+#
+# WARNING
+# ^^^^^^^
+# This configuration file is for development only and should NOT be used be
+# used in production. For example configuration, see the Presto documentation.
+#
+
+connector.name=hive-hadoop2
+hive.metastore.uri=thrift://hadoop-master:9083
+hive.metastore.thrift.client.socks-proxy=hadoop-master:1180
+hive.allow-drop-table=true
+hive.allow-rename-table=true
+hive.metastore-cache-ttl=0s
+hive.allow-add-column=true
+hive.allow-drop-column=true
+hive.allow-rename-column=true
+
+hive.metastore.authentication.type=KERBEROS
+hive.metastore.service.principal=hive/hadoop-master@LABS.TERADATA.COM
+hive.metastore.client.principal=hive-other/hadoop-master@OTHERLABS.TERADATA.COM
+hive.metastore.client.keytab=/etc/hive/conf/hive-other.keytab
+hive.config.resources = /etc/hadoop/conf/core-site.xml
+
+hive.hdfs.authentication.type=KERBEROS
+hive.hdfs.impersonation.enabled=false
+hive.hdfs.presto.principal=hdfs/hadoop-master@LABS.TERADATA.COM
```

Comment:
We need to have `hdfs` in `OTHERLABS.TERADATA.COM` to verify the cross realm

Reply:
@arhimondr Thanks for reviewing. 

Both the Modules `kerberosHiveMetastoreAuthenticationModule` and `kerberosHdfsAuthenticationModule` use `createCachingKerberosHadoopAuthentication` method in order to create `HadoopAuthentication` object with Kerberos. The workflow for authentication remains the same, since both of them use `HadoopAuthentication` object created in the same manner.

That's why I figured that testing one of them should suffice to verify the correctness of this PR. Should I still add an hdfs principal in other realm as well?